frontend/cypress/e2e/side_window/logbook_prior_notification_form/behavior.spec.ts

```diff
@@ -36,10 +37,17 @@ context('Side Window > Logbook Prior Notification Form > Behavior', () => {
 
     // Reset
     const operationDate = dayjs().subtract(6, 'hours').toISOString()
-    cy.request('PUT', `/bff/v1/prior_notifications/logbook/FAKE_OPERATION_116?operationDate=${operationDate}`, {
-      body: {
-        note: null
-      }
+    getAuthorizationHeader().then(authorization => {
```

Comment:
Je n'avais pas pensé à ça, c'est propre comme ça. J'ai hésité à te proposer de carrément avoir un custom `requestWithAuth()` mais ça ferait passer un `this.apply/call` + `arguments` vu le polymorphisme nécessaire donc ça ne vaut pas vraiment l'effort je pense.

Reply:
Oui j'ai hésité à wrapper `request` aussi, mais là c'est simple et rapide à comprendre également